ndohyep/templates/core/tags/section_listing_homepage.html

```diff
@@ -11,10 +11,19 @@
     <h2><a href="{% pageurl section %}">{{section.title}}</a></h2>
     <p>{{section.description}} </p>
         {% for page in section.featured_articles_in_homepage %}
-            <div class="article-link">
-            <h3><a href="{% pageurl page %}">{{page.title}} ></a></h3>
-            <a href="" class="comments-count">12</a>
-            </div><!-- /article-link -->
+            {% if request.user.is_authenticated %}
+                {% if request.user.profile.get_age.age_range == page.get_children.first.title %}
+                    <div class="article-link">
+                    <h3><a href="{% pageurl page %}">{{page.title}}</a></h3>
+                    <a href="" class="comments-count">12</a>
```

Comment:
Simon, It seems this code is there in develop branch. Please let us know whether we should get the latest code from develop branch in order to update in this branch.